api/scheduling/v1alpha1/clustersizingconfiguration_types.go

```diff
@@ -177,6 +181,24 @@ type TransitionDelayConfiguration struct {
 	Decrease metav1.Duration `json:"decrease,omitempty"`
 }
 
+type ResourceRequest struct {
+	// +kubebuilder:validation:Required
+	// DeploymentName is the name of the deployment to which the resource request applies.
+	DeploymentName string `json:"deploymentName"`
+
+	// +kubebuilder:validation:Required
+	// ContainerName is the name of the container to which the resource request applies.
+	ContainerName string `json:"containerName"`
+
+	// +kubebuilder:validation:Optional
+	// Memory is the amount of memory to request for the container.
+	Memory *resource.Quantity `json:"memory,omitempty"`
```

Comment:
This field can also be used to configure KAS, right? can we remove/deprecate `KASMemoryRequest` field, then? https://github.com/openshift/hypershift/blob/a72e5814d121aeadc03d573b3c714c6aac7d84e8/api/scheduling/v1alpha1/clustersizingconfiguration_types.go#L106

Reply:
Yup, makes sense. AFAIK we're not using the KASMemoryRequest field, so I will just remove it. Thx!

Reply:
Fixed